src/eq_schema/Questionnaire.js

```diff
@@ -3,10 +3,13 @@ const Group = require("./Group");
 
 class Questionnaire {
   constructor(authorJson) {
+    const questionnaireId = authorJson.id;
+    this.eq_id = questionnaireId;
+    this.form_type = questionnaireId;
```

Comment:
Schema needs updating to permit these values

Reply:
PR raised on Survey Runner to capture this https://github.com/ONSdigital/eq-survey-runner/pull/1298